packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -24,6 +25,7 @@ import { DEFAULT_CONFIG, MeterConfig } from './types';
  */
 export class MeterProvider implements types.MeterProvider {
   private readonly _meters: Map<string, Meter> = new Map();
+  readonly resource: Resource = Resource.createTelemetrySDKResource();
   readonly logger: types.Logger;
 
   constructor(private _config: MeterConfig = DEFAULT_CONFIG) {
```

Comment:
What do you think about making `resource` part of [MeterConfig](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-metrics/src/types.ts#L55-L67) with [default value ](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-metrics/src/types.ts#L70)either to `Resource.empty()` or `Resource.createTelemetrySDKResource()`? This way  users (and independent developers) can customize the resource attributes.

Reply:
I went with `Resource.createTelemetrySDKResource()` as the default. When we have resource auto detection, it will be updated to use that (e.g. `Resource.detectAll()` or similar).